ui/components/Navigator.js

```diff
@@ -665,35 +697,45 @@ class Navigator extends React.Component {
               );
             })}
             <Divider className={classes.divider} />
-            <ListItem
-              component="a"
-              href="https://meshery.io/"
-              target="_blank"
-              key="about"
-              className={classNames(classes.item, classes.itemActionable, classes.community)}
-            >
-              <div className={classNames(classes.link)}>
-                <Tooltip
-                  title="Community"
-                  placement="right"
-                  disableFocusListener={!isDrawerCollapsed}
-                  disableHoverListener={!isDrawerCollapsed}
-                  disableTouchListener={!isDrawerCollapsed}
-                >
-                  <ListItemIcon className={classes.listIcon}>
-                    <FontAwesomeIcon icon={faExternalLinkAlt} transform="shrink-2" fixedWidth />
-                  </ListItemIcon>
-                </Tooltip>
-                <ListItemText
-                  className={isDrawerCollapsed ? classes.isHidden : classes.isDisplayed}
-                  classes={{
-                    primary: classes.itemPrimary,
-                  }}
-                >
-                  Community
-                </ListItemText>
-              </div>
-            </ListItem>
+            {externlinks.map(({ id, icon, title, href }) => {
+              return (
+                <React.Fragment key={id}>
```

Comment:
React.Fragment is not needed here as anyways we are returning a single component here which is <ListItem>

Reply:
Oh I see! Will fix that :)